src/Symfony/Component/Console/Command/SignalableCommandInterface.php

```diff
@@ -15,6 +15,8 @@
  * Interface for command reacting to signal.
  *
  * @author Grégoire Pineau <lyrixx@lyrix.info>
+ *
+ * @method ?int getExitCode(int $signal, ?int $previousExitCode) Returns whether to automatically exit with $exitCode after command handling, or not with null.
```

Comment:
Me too! But I would love the following API:
```
public function handleSignal(int $signal, ?int &previousExitCode = null): ?int;
```

where you must to return the new exit code, but this is a BC break.

With your solution, we can avoid the BC break, but using a reference is not really common, and I (we?) tend to avoid it.

Is there a way to change the return type of an interface? (adding a new method and deprecated this one is the only way, isnt't?)